cmd/tsbs_load_clickhouse/creator.go

```diff
@@ -0,0 +1,235 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"strings"
+
+	"github.com/jmoiron/sqlx"
+)
+
+// loader.DBCreator interface implementation
+type dbCreator struct {
+	tags    string
+	cols    []string
+	connStr string
+}
+
+// loader.DBCreator interface implementation
+func (d *dbCreator) Init() {
+	br := loader.GetBufferedReader()
+	d.readDataHeader(br)
+}
+
+// readDataHeader fills dbCreator struct with data structure (tables description)
+// specified at the beginning of the data file
+func (d *dbCreator) readDataHeader(br *bufio.Reader) {
+	// First N lines are header, with the first line containing: "tags:comma-separated list of tags"
```

Comment:
This comment leads me to believe the first line is:
```
tags:hostname,region,datacenter,...
```
rather than:
```
tags,hostname,region,datacenter,...
```

(same with the description of second through N-1 lines)